Modules/RaidManager/RaidManager.lua

```diff
@@ -282,6 +283,14 @@ function RaidManager:StartRaid(raid)
         end
     end
 
+    -- Auto reward on-time bonus if set
+    if raid:Configuration():Get("raidCompletionBonus") then
+        LOG:Debug(string.format("[%s]: [%s]", CONSTANTS.POINT_CHANGE_REASONS.GENERAL[CONSTANTS.POINT_CHANGE_REASON.ON_TIME_BONUS], raid:Configuration():Get("onTimeBonusValue")))
+        PointManager:UpdatePoints(raid:Roster(), players, raid:Configuration():Get("onTimeBonusValue"), CONSTANTS.POINT_CHANGE_REASON.ON_TIME_BONUS, 0, true)
```

Comment:
I would suggest using new type of ledger entry: `award points to roster`. this will make it way easier. also it needs to be added after the raid status is updated

Reply:
This should be also moved **after** line 294 (submiting of raid entry) to have the cause-effect continuity